### PR TITLE
replace findbugs plugin with spotbugs plugin

### PR DESCRIPTION
**Pull request by SingingBush**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -397,6 +397,7 @@
         </configuration>
       </plugin>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```
Comment (garydgregory) [inline pom.xml:400]: Hello @SingingBush ,
Thank you for your PR.
Please keep your chanes to the minimum and focused. These types of changes are not required.

**Review by garydgregory — CHANGES_REQUESTED** [review]
See comment.

File: pom.xml (hunk 1 above)
Comment (SingingBush) [reply pom.xml:400]: no problem, fixing now

**Comment by garydgregory** [comment]
Closing: Done in a separate PR.

**Pull request closed without merging**